### DISCUSSION TITLE
How to add webp support for ImageMagick?

### DISCUSSION THREAD

**Discussion by plutons** [ImageMagick/ImageMagick#5812] [Help] [answered]
First I download libwebp and compile install it
`
wget https://download.imagemagick.org/archive/delegates/libwebp-0.6.0.tar.gz
tar -zxvf libwebp-0.6.0.tar.gz
cd libwebp-0.6.0
./configure
make && make install
`
Second I download ImageMagick and compile install it
`
wget https://imagemagick.org/archive/ImageMagick.tar.gz
tar -zxvf ImageMagick.tar.gz
cd ImageMagick
./configure --prefix=/usr/local/ImageMagick --with-webp=yes
`
the log is 
Delegate library configuration:
  WEBP              --with-webp=yes		no
  WMF               --with-wmf=no		no
  X11               --with-x=			no
  XML               --with-xml=yes		no
  ZIP               --with-zip=yes		no
  ZLIB              --with-zlib=yes		yes
  ZSTD              --with-zstd=yes		no
How to add webp support for ImageMagick?

**Comment by urban-warrior** [comment, marked as answer]
Check `config.log` to determine why the Webp delegate library failed to validate. We suspect its because Webp was installed in the `/usr/local` path.  If so, you may need to point the package configuration to that location.  For example,

```
$ export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
$ ./configure
```

**Reply by plutons** [reply]
Thank you！！！
It works
